v4/core/detailed_response.go

```diff
@@ -91,5 +91,5 @@ func (response *DetailedResponse) String() string {
 	if err == nil {
 		return fmt.Sprintf("%+v\n", string(output))
 	}
-	return fmt.Sprintf("Response")
```

Comment:
I saw a linter error on this statement due to the unnecessary use of fmt.Sprintf().   This is the first go core PR I've worked on after upgrading my laptop to Fedora 32 (the latest) which uses go version 1.14.3.   Prior to this, I was running Fedora 30, which uses go 1.12.x.   I'm guessing that the upgrade to the latest Go engine is why I'm now seeing this linter error, when it wasn't seen before.   Anyway, I fixed it :)